src/test/resources/org/danilopianini/gradle/test/ktmultiplatform/test.yaml

```diff
@@ -2,7 +2,8 @@ tests:
   - description: "gradle should generate metadata files in kotlin-multiplatform projects"
     configuration:
       tasks: &tasks
-        - 'generateMetadataFileForJavaOSSRHPublication'
+#        # adding this task makes the test fail
+#        - 'generateMetadataFileForJavaOSSRHPublication'
```

Comment:
Why? This task must be working

Reply:
Dunno. I've simplified a bit the test and the task is gone.
I'm supposing it's a matter of updated Kotlin version: the test was using Kotlin 1.7.x, while I'm now using the same version used for development (1.8.x).

I summon @nicolasfara, 'cause I believe that task is useless for MP projects, but let's discuss

Reply:
@nicolasfara tell us something

Reply:
Sorry guys for the delay. I've investigated and apparently the task `generateMetadataFileForJavaOSSRHPublication` has been gone since kotlin 1.8.x.
Also in my opinion that task is not useful in kotlin-MP projects